includes/classes/blocks/class-query-loop.php

```
@@ -89,6 +89,34 @@ public function maybe_hide_varitaion( $block_content, $parsed_block, $block_obj
 		}
 
 		if ( in_array( 'travel-information', $matches ) ) {
+			// Check if ANY travel information fields have content
+			$travel_info_keys = array(
```

PHPMD (2.15.0)

The variable $travel_info_keys is not named in camelCase. (undefined)

(CamelCaseVariableName)